### DISCUSSION TITLE
Add sum function to OQL?

### DISCUSSION THREAD

**Discussion by pims848** [Security-Onion-Solutions/securityonion#9927] [General]
Hi team,
Great work you're doing here !
I was analyzing some large PCAP volume with SO analyst VM and I needed to find top talkers quickly.
I could create a Kibana dashboard for that but I was wondering if we could have this in Dashboard or Hunt view maybe?
Thx

**Comment by InfosecGoon** [comment]
Thanks, I'll pass that along to the developers!
